### PR TITLE
refactor(console): add sign in method button

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

- add a button state when no sign-in method is selected.
- rename `AddSignInMethodButton` -> `AddButton` for it is under the `SignInMethodEditBox` scope.

## Testing

image

### CODE REVIEW

File: packages/phrases/src/locales/en/translation/admin-console/sign-in-exp.ts [hunk 1]
```diff
@@ -58,6 +58,7 @@ const sign_in_exp = {
       sign_in_identifier_and_auth: 'Sign in identifier and authentication',
       description:
         'Users can use any one of the selected ways to sign in. Drag and drop to define identifier priority regarding the sign in flow. You can also define the password or verification code priority.',
+      add_sign_in_method: 'Add Sign In Method',
```
Comment (charIeszhao) [inline packages/phrases/src/locales/en/translation/admin-console/sign-in-exp.ts:61]:
```suggestion
      add_sign_in_method: 'Add Sign-in Method',
```

**Comment by charIeszhao** [comment]
image

Can the dropdown menu be the same width as the button?

File: packages/console/src/pages/SignInExperience/tabs/SignUpAndSignInTab/components/SignInMethodEditBox/AddButton.tsx [hunk 2]
```diff
@@ -2,30 +2,41 @@ import type { SignInIdentifier } from '@logto/schemas';
...
 };
 
-const AddSignInMethodButton = ({ options, onSelected }: Props) => {
+const AddButton = ({ options, onSelected, hasSelectedIdentifiers }: Props) => {
   const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
 
   if (options.length === 0) {
     return null;
   }
+  const addSocialConnectorButtonProps: ButtonProps = {
```
Comment (charIeszhao) [inline packages/console/src/pages/SignInExperience/tabs/SignUpAndSignInTab/components/SignInMethodEditBox/AddButton.tsx:24]:
```suggestion
  const addSignInMethodButtonProps: ButtonProps = {
```

File: packages/phrases/src/locales/en/translation/admin-console/sign-in-exp.ts (hunk 1 above)
Comment (xiaoyijun) [reply packages/phrases/src/locales/en/translation/admin-console/sign-in-exp.ts]: updated.

**Review by charIeszhao — APPROVED** [review]

**Pull request merged**